httpx/_compat.py

```diff
@@ -13,12 +13,22 @@
     from async_generator import asynccontextmanager  # type: ignore # noqa
 
 
-def set_minimum_tls_version_1_2(context: ssl.SSLContext) -> None:
-    if sys.version_info >= (3, 10):
+if sys.version_info >= (3, 10) or (
```

Comment:
So there's still a little too much unnecessary change footprint here. We don't really need to switch the structure around here. Let's keep `set_minimum_tls_version_1_2` as the top level function, with the switch *inside* it.

We really just want to:

* Update the `sys.version_info >= (3, 10)` branch condition.
* Possibly neaten up the commenting.